h2o-core/src/main/java/hex/ModelMetricsBinomial.java

```diff
@@ -74,6 +74,8 @@ public ConfusionMatrix cm(AUC2.ThresholdCriterion criterion) {
 
   // expose simple metrics criteria for sorting
   public double auc() { return auc_obj()._auc; }
+  public double pr_auc() { return auc_obj()._pr_auc; }
+  public double aucpr() { return auc_obj()._pr_auc; } // for compatibility with naming in ScoreKeeper.StoppingMetric annotation
```

Comment:
should we also add the `pr_auc` one?
`aucpr` is needed for the reflection logic in `ScoringInfo`

Reply:
@seb-h2o Are there advantages to adding `pr_auc()` other than consistency (all the other clients have both?).

Reply:
other than consistency? no, as it was exposed only through REST API, which doesn't need those methods.
R currently has only `pr_auc` and Python has  `model.pr_auc()` though...